Makefile

```diff
@@ -1,4 +1,4 @@
-SEMANTIC_VER=0.2.1+
+SEMANTIC_VER=0.3.1+
```

Comment:
Would you consider a revision change to 0.2.2... since the addition of a field to the customer is not backwards breaking?

Reply:
yes!